text-embeddings-inference/custom_server/.maintain/roll_out_docker.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
do we want to keep this?

Reply:
Not sure, that's why I left it in. Do we want to keep this?